test/common/utils.js

```diff
@@ -6,7 +6,10 @@ export function randomFiscalCode() {
     const name = randomString(6)
     const birth_y = (40 + Math.floor(Math.random() * 50)).toString()
```

Comment:
Maybe we could use a random JavaScript Date object here

Reply:
done, thanks :)